core/src/main/java/io/micronaut/core/type/Argument.java

```diff
@@ -381,12 +437,14 @@ default boolean isAssignableFrom(@NonNull Argument<?> candidateArgument) {
 
     /**
      * Allows coercing a {@link Type} instance to an {@link Argument}.
+     *
      * @param type The type
      * @return The argument
      * @throws IllegalArgumentException If the type cannot be coerced
      * @since 3.0.0
      */
-    static @NonNull Argument<?> of(@NonNull Type type) {
+    static @NonNull
+    Argument<?> of(@NonNull Type type) {
```

Comment:
should this have an implementation for TypeVariable, now?

Reply:
Perhaps, but our runtime reconstruction from reflection data is largely missing and not sure we need it or want it